### ISSUE TITLE
KeyError: Can't build contract

### ISSUE THREAD

**Issue by LevZed** [sky-ring/rift#9]
Following the instructions in the README, I initialize the project and try to build it. As a result, I get a keyError.
Rift was installed from source.
After changing the generated base contract, I get the same error.

Here is the output in the console:
```
C:\Users\Lev\dev\TON\rift_tests\rift-f-test>rift build
Building rift-f-test project ...
compiling rift-f-test.py
Traceback (most recent call last):
  File "C:\Users\Lev\AppData\Local\Programs\Python\Python310\lib\runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "C:\Users\Lev\AppData\Local\Programs\Python\Python310\lib\runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "C:\Users\Lev\AppData\Local\Programs\Python\Python310\Scripts\rift.exe\__main__.py", line 7, in <module>
  File "C:\Users\Lev\AppData\Local\Programs\Python\Python310\lib\site-packages\click\core.py", line 1130, in __call__
    return self.main(*args, **kwargs)
  File "C:\Users\Lev\AppData\Local\Programs\Python\Python310\lib\site-packages\click\core.py", line 1055, in main
    rv = self.invoke(ctx)
  File "C:\Users\Lev\AppData\Local\Programs\Python\Python310\lib\site-packages\click\core.py", line 1657, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "C:\Users\Lev\AppData\Local\Programs\Python\Python310\lib\site-packages\click\core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "C:\Users\Lev\AppData\Local\Programs\Python\Python310\lib\site-packages\click\core.py", line 760, in invoke
    return __callback(*args, **kwargs)
  File "C:\Users\Lev\dev\TON\rift\rift\cli\commands\build.py", line 93, in build
    contracts = sorted(contracts, key=lambda c: c_order[c.__module__])
  File "C:\Users\Lev\dev\TON\rift\rift\cli\commands\build.py", line 93, in <lambda>
    contracts = sorted(contracts, key=lambda c: c_order[c.__module__])
KeyError: 'rift-f-test'
```

**Comment by LevZed** [comment]
Might be a mistake on my part, so I'll check on another system later.

**Comment by AminRezaei0x443** [comment]
It's fixed, please check out the latest version and tell me whether it's ok for you or not.

**Comment by LevZed** [comment]
Yes, now `build` works! 👍

**Comment by LevZed** [comment]
Fixed in 1e4ffa1efda1906ee4a9dbb772668422861d8cad
